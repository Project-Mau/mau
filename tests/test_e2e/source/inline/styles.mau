This paragraph contains _underscore_ text.

This paragraph contains *star* text.

This paragraph contains ~tilde~ text.

This paragraph contains ^caret^ text.
