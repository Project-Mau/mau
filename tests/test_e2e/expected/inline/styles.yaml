data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'This paragraph contains '
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: underscore
          kwargs: {}
          subtype: null
          tags: []
          type: style
          value: underscore
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: ' text.'
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'This paragraph contains '
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: star
          kwargs: {}
          subtype: null
          tags: []
          type: style
          value: star
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: ' text.'
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'This paragraph contains '
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: tilde
          kwargs: {}
          subtype: null
          tags: []
          type: style
          value: tilde
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: ' text.'
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'This paragraph contains '
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: caret
          kwargs: {}
          subtype: null
          tags: []
          type: style
          value: caret
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: ' text.'
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

